views/templates/steps/post-update.html.twig

```diff
@@ -85,7 +85,7 @@
 
     {% if download_logs is defined %}
     <div class="content__section">
-      <a class="link" href="{{ download_logs.download_path }}" download="{{ download_logs.filename }}">
+      <a class="link" href="{{ download_logs.download_path }}" download="{{ download_logs.filename }}" data-au-tracking="Update logs downloaded">
```

Comment:
These events could reuse the constants defined in a PHP class, to avoid duplicating the same event name in different places (Currently duplicated in LogsService.php)